### PR TITLE
Allow developer to specify e2e test name on cli

### PR DESCRIPTION
**Pull request by weshayutin**
## Why the changes were made

BECAUSE, you shouldn't have to change src code to specify the dam test you want to execute.
GOLANG / Ginkgo pffft.

## How to test the changes made

make test-e2e TEST_ARGS="--ginkgo.focus='MySQL application DATAMOVER'"
should only run 1 test :)

### CODE REVIEW

**Review by mrnold — APPROVED** [review]

**Review by shubham-pampattiwar — DISMISSED** [review]

File: Makefile [hunk 1]
```diff
@@ -526,7 +526,8 @@ test-e2e: test-e2e-setup install-ginkgo
 	--ginkgo.vv \
 	--ginkgo.no-color=$(OPENSHIFT_CI) \
 	--ginkgo.label-filter="$(TEST_FILTER)" \
-	--ginkgo.timeout=2h
+	--ginkgo.timeout=2h \
+	$(TEST_ARGS)
```
Comment (kaovilai) [inline Makefile:530]:
```suggestion
	$(GINKGO_ARGS)
```
?

Comment (weshayutin) [reply Makefile]: good suggestion, thank you @kaovilai

**Review by kaovilai — DISMISSED** [review]
/lgtm

**Review by mateusoliveira43 — CHANGES_REQUESTED** [review]
TEST_ARGS -> GINKGO_ARGS

File: docs/developer/testing/TESTING.md [hunk 2]
```diff
@@ -87,6 +87,12 @@ FIt("the assertion", func() { ... })
 ...
 ```
 
+You can also execute make test-e2e with a $TEST_ARGS variable set. Example:
```
[the remark this answers is not in the document]
Comment (weshayutin) [reply docs/developer/testing/TESTING.md]: ah dang, updated and also corrected the note about changing the code to specify which test to run.

**Review by mateusoliveira43 — APPROVED** [review]

**Review by shubham-pampattiwar — APPROVED** [review]

**Pull request merged**